engine/Engine_Dronecaster.sc

```diff
@@ -31,7 +31,13 @@ Dronecaster {
 
 	start { arg name;
 		if (drones.keys.includes(name), {
-			socket.setSource(drones[name]);
+			socket.setSource({
+			    arg hz=440, amp=0.02, amplag=0.02, hzlag=0.01;
+			    var amp_, hz_;
+			    amp_ = Lag.ar(K2A.ar(amp), amplag);
+			    hz_ = Lag.ar(K2A.ar(hz), hzlag);
+			    drones[name].value(hz_,amp_);
```

Comment:
Would these still work as named arguments? Kind of a nitpick but it might be a good future-proofing measure.

Reply:
do you mean, is it possible to do `drones[name].value(hz=hz_,amp=amp_)`? if so- I don't the answer and will try and see.